app/services/proposal_pdf_service.rb

```diff
@@ -603,6 +603,23 @@ def invites_gender_data
   def proposal_supplementary_files
     @text << "\\pagebreak"
     @text << "\\section*{Supplementry Files}\n\n"
+    @text << "\n\\begin{figure}\n  \\centering\n"
+
+    @proposal.files&.each do |file|
+      path = ActiveStorage::Blob.service.send(:path_for, file.key)
+      raise "Error: #{file.filename} is not a PDF!" unless File.extname(file.filename) == /\.pdf$/i
+
+      file_content = File.read(path)
+      file_name = "#{@proposal&.code}-#{file.filename}"
+      write_attachment_file(file_content, file_name)
+      @text << "    \\includegraphics{#{file_name}}\n"
```

Comment:
@brentkearney5 - I did same as you mentioned. But got an error and I didn't get any solution
![package_error](https://user-images.githubusercontent.com/45980144/135471203-047e3b65-acad-4e89-8a22-6821d62e12e2.png)
.

Reply:
Was that file actually written to the tmp directory?

Does it work with a file without a space in the name?